### PR TITLE
Remove IcoMoon fonts from the metro theme

### PR DESCRIPTION
**Pull request by williamdes**
I struggle with packaging 5.0 into Debian and 5.1 will be next.
I need that our situation is clean and clear for me to be able to make packaging decisions for themes.

So I removed IcoMoon because it was more or less useless.

I will have to clarify the `themes/metro/fonts/opensans-*` files to know where they come from.

This PR is all about making me able to have reproductible builds from source and now metro theme is a nightmare..
This needs to enter QA_5_1 very quickly.

### CODE REVIEW

**Comment by williamdes** [comment]
Draft because I am not finished, please do let me know your thoughts

**Comment by williamdes** [comment]
:warning: I changed the icon from (#15912 - Fix #14953) because I can not get it for free on their website and the license is unclear

**Comment by mauriciofauth** [comment]
Is it not possible to completely remove the Metro theme for 5.1 Debian package?

**Comment by williamdes** [comment]
> Is it not possible to completely remove the Metro theme for 5.1 Debian package?

Good question, it is possible but I am afraid to have complaints and bugs filed to me and have to add it back with just the same amount of work

File: doc/vendors.rst [hunk 1]
```diff
@@ -30,3 +30,21 @@ js/vendor
...
+- `themes/*/img/b_rename.svg` Is a Material Design Icon, the `form-textbox <https://materialdesignicons.com/icon/form-textbox>`_
+- `themes/metro/img/cancel_circle.svg` Is a IcoMoon the `cancel-circle <https://github.com/Keyamoon/IcoMoon-Free/blob/master/SVG/270-cancel-circle.svg>`_
+- `themes/metro/img/user.svg` Is a IcoMoon the `user <https://github.com/Keyamoon/IcoMoon-Free/blob/master/SVG/144-user.svg>`_
+
+CC BY 4.0 or GPL
+
+Licenses for vendors
+--------------------
+
+- Material Design Icons is under the `"Pictogrammers Free License" <https://github.com/Templarian/MaterialDesign/blob/master/LICENSE>`_ (Apache 2.0).
```
Comment (mauriciofauth) [inline doc/vendors.rst:48]: Apache 2 is not compatible with GPLv2. See: https://www.gnu.org/licenses/license-list.html#apache2

Comment (williamdes) [reply doc/vendors.rst:48]: Thanks for the hint, I will fix that

**Comment by williamdes** [comment]
Hi @mauriciofauth 

You said 

> Apache 2 is not compatible with GPLv2. See: https://www.gnu.org/licenses/license-list.html#apache2

Seems like the opensans fonts we have are under Apache too but we have them in our code base
https://fonts.google.com/specimen/Open+Sans#license

https://github.com/phpmyadmin/phpmyadmin/tree/master/themes/metro/fonts

Am I mistaken or are the opensans not opensans files but names as opensans files :thinking: 

This metadata leads me to think the files are Apache licenced, the metadata says so

```
$ otfinfo --info themes/metro/fonts/*.ttf
themes/metro/fonts/opensans-bold-webfont.ttf:Family:              Open Sans
themes/metro/fonts/opensans-bold-webfont.ttf:Subfamily:           Bold
themes/metro/fonts/opensans-bold-webfont.ttf:Full name:           Open Sans Bold
themes/metro/fonts/opensans-bold-webfont.ttf:PostScript name:     OpenSans-Bold
themes/metro/fonts/opensans-bold-webfont.ttf:Version:             Version 1.10
themes/metro/fonts/opensans-bold-webfont.ttf:Unique ID:           Ascender - Open Sans Bold Build 100
themes/metro/fonts/opensans-bold-webfont.ttf:Designer URL:        http://www.ascendercorp.com/typedesigners.html
themes/metro/fonts/opensans-bold-webfont.ttf:Manufacturer:        Ascender Corporation
themes/metro/fonts/opensans-bold-webfont.ttf:Vendor URL:          http://www.ascendercorp.com/
themes/metro/fonts/opensans-bold-webfont.ttf:Trademark:           Open Sans is a trademark of Google and may be registered in certain jurisdictions.
themes/metro/fonts/opensans-bold-webfont.ttf:Copyright:           Digitized data copyright © 2010-2011, Google Corporation.
themes/metro/fonts/opensans-bold-webfont.ttf:License URL:         http://www.apache.org/licenses/LICENSE-2.0
themes/metro/fonts/opensans-bold-webfont.ttf:License Description: Licensed under the Apache License, Version 2.0
themes/metro/fonts/opensans-bold-webfont.ttf:Vendor ID:           1ASC
themes/metro/fonts/opensans-extrabold-webfont.ttf:Family:              Open Sans Extrabold
themes/metro/fonts/opensans-extrabold-webfont.ttf:Subfamily:           Regular
themes/metro/fonts/opensans-extrabold-webfont.ttf:Full name:           Open Sans Extrabold Regular
themes/metro/fonts/opensans-extrabold-webfont.ttf:PostScript name:     OpenSans-Extrabold
themes/metro/fonts/opensans-extrabold-webfont.ttf:Preferred family:    Open Sans
themes/metro/fonts/opensans-extrabold-webfont.ttf:Preferred subfamily: Extrabold
themes/metro/fonts/opensans-extrabold-webfont.ttf:Version:             Version 1.10
themes/metro/fonts/opensans-extrabold-webfont.ttf:Unique ID:           Ascender - Open Sans Extrabold Build 100
themes/metro/fonts/opensans-extrabold-webfont.ttf:Designer URL:        http://www.ascendercorp.com/typedesigners.html
themes/metro/fonts/opensans-extrabold-webfont.ttf:Manufacturer:        Ascender Corporation
themes/metro/fonts/opensans-extrabold-webfont.ttf:Vendor URL:          http://www.ascendercorp.com/
themes/metro/fonts/opensans-extrabold-webfont.ttf:Trademark:           Open Sans is a trademark of Google and may be registered in certain jurisdictions.
themes/metro/fonts/opensans-extrabold-webfont.ttf:Copyright:           Digitized data copyright © 2011, Google Corporation.
themes/metro/fonts/opensans-extrabold-webfont.ttf:License URL:         http://www.apache.org/licenses/LICENSE-2.0
themes/metro/fonts/opensans-extrabold-webfont.ttf:License Description: Licensed under the Apache License, Version 2.0
themes/metro/fonts/opensans-extrabold-webfont.ttf:Vendor ID:           1ASC
themes/metro/fonts/opensans-light-webfont.ttf:Family:              Open Sans Light
themes/metro/fonts/opensans-light-webfont.ttf:Subfamily:           Regular
themes/metro/fonts/opensans-light-webfont.ttf:Full name:           Open Sans Light Regular
themes/metro/fonts/opensans-light-webfont.ttf:PostScript name:     OpenSans-Light
themes/metro/fonts/opensans-light-webfont.ttf:Preferred family:    Open Sans
themes/metro/fonts/opensans-light-webfont.ttf:Preferred subfamily: Light
themes/metro/fonts/opensans-light-webfont.ttf:Version:             Version 1.10
themes/metro/fonts/opensans-light-webfont.ttf:Unique ID:           Ascender - Open Sans Light Build 100
themes/metro/fonts/opensans-light-webfont.ttf:Designer URL:        http://www.ascendercorp.com/typedesigners.html
themes/metro/fonts/opensans-light-webfont.ttf:Manufacturer:        Ascender Corporation
themes/metro/fonts/opensans-light-webfont.ttf:Vendor URL:          http://www.ascendercorp.com/
themes/metro/fonts/opensans-light-webfont.ttf:Trademark:           Open Sans is a trademark of Google and may be registered in certain jurisdictions.
themes/metro/fonts/opensans-light-webfont.ttf:Copyright:           Digitized data copyright © 2010-2011, Google Corporation.
themes/metro/fonts/opensans-light-webfont.ttf:License URL:         http://www.apache.org/licenses/LICENSE-2.0
themes/metro/fonts/opensans-light-webfont.ttf:License Description: Licensed under the Apache License, Version 2.0
themes/metro/fonts/opensans-light-webfont.ttf:Vendor ID:           1ASC
themes/metro/fonts/opensans-regular-webfont.ttf:Family:              Open Sans
themes/metro/fonts/opensans-regular-webfont.ttf:Subfamily:           Regular
themes/metro/fonts/opensans-regular-webfont.ttf:Full name:           Open Sans Regular
themes/metro/fonts/opensans-regular-webfont.ttf:PostScript name:     OpenSans
themes/metro/fonts/opensans-regular-webfont.ttf:Version:             Version 1.10
themes/metro/fonts/opensans-regular-webfont.ttf:Unique ID:           Ascender - Open Sans Build 100
themes/metro/fonts/opensans-regular-webfont.ttf:Designer URL:        http://www.ascendercorp.com/typedesigners.html
themes/metro/fonts/opensans-regular-webfont.ttf:Manufacturer:        Ascender Corporation
themes/metro/fonts/opensans-regular-webfont.ttf:Vendor URL:          http://www.ascendercorp.com/
themes/metro/fonts/opensans-regular-webfont.ttf:Trademark:           Open Sans is a trademark of Google and may be registered in certain jurisdictions.
themes/metro/fonts/opensans-regular-webfont.ttf:Copyright:           Digitized data copyright © 2010-2011, Google Corporation.
themes/metro/fonts/opensans-regular-webfont.ttf:License URL:         http://www.apache.org/licenses/LICENSE-2.0
themes/metro/fonts/opensans-regular-webfont.ttf:License Description: Licensed under the Apache License, Version 2.0
themes/metro/fonts/opensans-regular-webfont.ttf:Vendor ID:           1ASC
```

cc @ibennetch

**Comment by mauriciofauth** [comment]
I think the best option is to remove the font files, but keep the `font-family: 'Open Sans, sans;` to use open sans if it's installed on the user system.

**Comment by williamdes** [comment]
> I think the best option is to remove the font files, but keep the `font-family: 'Open Sans, sans;` to use open sans if it's installed on the user system.

Very good idea I will push that, looks like this will be a great patch for Debian
Thanks a lot!

Do you think I should sync back the theme to the themes repository afterwards?

**Comment by mauriciofauth** [comment]
I think you can remove in the phpMyAdmin repository, instead of creating a
patch for Debian. Since the license is not compatible.

I don't know how we are handling the Metro theme on the themes repo.

Em qui, 24 de dez de 2020 21:34, William Desportes <notifications@github.com>
escreveu:

**Comment by williamdes** [comment]
Sure
I just need to patch 5.0 on Debian
I can not upgrade to 5.1 for now, it will take me more time

Anyway I will push some more commits here but if you feel like it you can push commits to fix the user icon on the metro login page here

File: doc/vendors.rst (hunk 1 above)
Comment (williamdes) [reply doc/vendors.rst:48]: @mauriciofauth does this rename Icon see legit to you https://thenounproject.com/term/rename/61456/ ?
I am afraid it is not public domain but copied, what do you think ?

File: doc/vendors.rst [hunk 2]
```diff
@@ -30,3 +30,21 @@ js/vendor
...
     dependencies.
+
+Specific files LICENSES
+-----------------------
+
+phpMyAdmin distributed themes contain some content that is under licenses.
+
+- `themes/*/img/b_rename.svg` Is a `Icons8 <https://thenounproject.com/Icons8/>`_, icon from the `Android L Icon Pack Collection <https://thenounproject.com/Icons8/collection/android-l-icon-pack/>`_. The icon `rename <https://thenounproject.com/term/rename/61456/>`_.
+- `themes/metro/img/cancel_circle.svg` Is a IcoMoon the `cancel-circle <https://github.com/Keyamoon/IcoMoon-Free/blob/master/SVG/270-cancel-circle.svg>`_
+- `themes/metro/img/user.svg` Is a IcoMoon the `user <https://github.com/Keyamoon/IcoMoon-Free/blob/master/SVG/144-user.svg>`_
```
Comment (mauriciofauth) [inline doc/vendors.rst:41]:
```suggestion
- `themes/metro/img/user.svg` Is a IcoMoon the `user <https://github.com/Keyamoon/IcoMoon-Free/blob/master/SVG/114-user.svg>`_
```

**Review by MauricioFauth — APPROVED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**